### ISSUE TITLE
Open "thread" modal in third column instead

### ISSUE THREAD

**Issue by afontenot** [thomaswang/minimal-twitter#10]
Following up from [HN](https://news.ycombinator.com/item?id=20522017), the following suggestion would result in wasting much less horizontal space (which Twitter doesn't seem interested in using for anything worthwhile).

I think a three-column format would be cool. Basically, the first column is the thin icons you have now. The second column is the tweet column. The third column (equal in size to the second column) would appear when you click on a tweet; instead of loading the responses to that tweet as a modal view as it does now, the responses would appear in the right column.

Basically, this would make Twitter a lot more like several email clients or an RSS reader. Quick and dirty mockup: ![screenshot](https://user-images.githubusercontent.com/3870410/61881589-dd3b8800-aec4-11e9-88eb-ec62dca8f103.png)

**Comment by JaxonWright** [comment]
Although I like the idea, this is the opposite of "minimal"

**Comment by thomaswang** [comment]
Not planning to add this for now, sorry!
